docs/EVG.md

```diff
@@ -1864,3 +1872,20 @@ guidelines:
    jurisdiction as specified in Section 9.2.4.  The stated address of the organisation 
    combined with the organization name SHALL NOT be the only information used to 
    disambiguate the organisation.
+   
+**LEI**: The Legal Entity Identifier (LEI) as specified in the ISO 17442 and registered
+   in the Global LEI Index.  
+   
+   1. This information SHALL be validated by matching the organization name and 
+   registration number found in the Global LEI Index against the Subject Organization 
+   Name Field (see 9.2.1) and Subject Registration Number Field (see 9.2.5) within the 
+   context of the subject’s jurisdiction as specified in Section 9.2.4.  
```

Comment:
It's completely undefined here what "matching" means, and this is where I wanted to flesh out more comprehensive normative requirements.

> Before including an LEI within a certificate, the CA SHALL
> * Confirm that the ValidationSources field of the associated LEI record, as reported by GLEIF, contains FULLY_CORROBORATED
> * That the RegistrationAuthority of the associated LEI Record is the same entity as the Incorporating Agency or Registration Agency used in validating the information within the Subject Jurisdiction of Incorporation or Registration Field (see Section 9.2.4) and the Subject Registration Number Field (see Section 9.2.5)
> * That the Subject Organization Name Field (see Section 9.2.1) is contained, exactly as entered, within either the LegalName, OtherEntityNames, or TransliteratedOtherEntityNames of the associated LEI Record
> * That the Subject Registration Number (see Section 9.2.5) is equal, exactly as entered, to the RegistrationAuthorityEntityID of the associated LEI Record

That said, I'm concerned about whether or not we need to worry about the Registration Details - in particular, the registration status - e.g. should a lapsed or canceled LEI Registration be used? What about challenges to the data source? Should we worry about the LegalJursidiction code and matching it?